src/main/webapp/drugs/marijuana.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- Drug Name here --> 
+    <title>Marijuana</title>
+    <link rel="stylesheet" href="project.style.css">
+</head>
+
+<body>
+    <!-- Name and aliases-->
+    <div id="drug name">
+        <h1>Marijuana </h1>
+        <i><b>Aliases: weed, pot, ganja, mary jane</b></i>
+    </div>
+    <!-- Emergency Info -->
+    <div id="emergency info">
+        <h3>Emergency Information</h3>
+        <p>
+            There have been no reported cases of people overdosing on marijuana alone and it is rarely laced with other drugs.
+            <br> Signs of laced marijuana include: 
+            <br>
+            <ul>
+                <li>Harsh Chemical Smell</li>
+                <li>The substance scratching surfaces when it is rubbed against them </li>
+                <li>Excessive amounts of powder/dust</li>
+            </ul> 
+            <br> In this case where you think marijuana you used was laced, seek medical attention immediately.
+        </p>
+    </div>
+    <!-- How the drug works-->
+    <div id="science behind drug">
+        <h3>How it works:</h3>
+        <p>
+            Marijuana is a plant that grows naturally. It contains THC, a chemical that is responsible for the symptoms of using the drug.
```

Comment:
It's worth thinking about whether this can be represented at some combination of plain text and lists (like a summary paragraph as a plain text string followed by a list of symptoms or whether we need support for something like markdown so that we can store formatted text on the server and render it client-side.